### ISSUE TITLE
Support for cultures with a non-point separator in a floating number.

### ISSUE THREAD

**Issue by FoggyFinder** [LayoutFarm/Typography#134]
In the [`double ReadRealNumber()`](https://github.com/LayoutFarm/Typography/blob/master/Typography.OpenFont/Tables.CFF/CFF.cs#L1288) method there is [using](https://github.com/LayoutFarm/Typography/blob/master/Typography.OpenFont/Tables.CFF/CFF.cs#L1332) of point as separator for double value.

```csharp
sb.Append(".");
```

that raises exception if current culture has another separator.

Maybe it better to rewrite the line to:

```csharp
sb.Append(System.Globalization.CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator);
```

or provide information about culture at `double.Parse`.

**Comment by Happypig375** [comment]
Or use InvariantCulture everywhere when parsing.

**Comment by prepare** [comment]
Thank you, fixed in 3e876b069c2d01b5cd12f5c92aa2a2853422a971
